### PR TITLE
My test cases 

### PR DESCRIPTION
**Pull request by 4stuntman4**

### CODE REVIEW

File: tests/Dmitry Tregubov/test2.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        // browser.pause(5000);
+    });
+
+    it ('Verify that you can change LF2 by typing an integer:1 (Boundary case)', () => {
+
+        //Pre-requisites: LF1 = 1; LF2 = 9
+        const clean = $('//input[@name=\'upper\']').clearValue();
+        const res = $('//input[@name=\'upper\']').setValue("1");
+
+        //  browser.pause(5000);
```
Comment (Anastasiia-Bandurova) [inline tests/Dmitry Tregubov/test2.js:18]: Delete pls this line
